### DISCUSSION TITLE
240329

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#90] [팀회의]
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항

## 어제 한 일
- 제가 맡은 모든 페이지 api 경로 수정 완료
- useMemo 리팩토링 시도하다가 우선 stash
- 자유게시판 페이지네이션 시작

## 오늘 할 일
- 자유게시판 페이지네이션
- 자유게시판 좋아요 기능

## 오늘의 일정 (코어 타임)
- 1:30 매니저님 면담

## 멘토님 질문 사항

**Reply by MoonPillGyeom** [reply]
와 저 어제 면담이였는데... 안 들어갔네요 ㅋㅋㅋㅋㅋㅋ

**Comment by MoonPillGyeom** [comment]
## 논의사항
일이 점점 줄어드는게 아니고 늘어나는거 같은데...어떻게하죠?ㅠ
## 어제 한 일
어제 사실 api가 자꾸 막혀서 많이 못했습니다..
## 오늘 할 일
열심히 하기!
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
x

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
userContext GET /user/ 요청 10번 가는 문제 + user 상태 없이 접근 시 login 리다이렉트
## 오늘 할 일
저거 다른 페이지에도 할 수 있으면 하고 그거 말고 없습니다.
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
